source/pgnloader.py

```diff
@@ -0,0 +1,14 @@
+import chess.pgn as pgnlib
+import pathlib
+
+
+class PgnFileLoader:
+
+    def __init__(self, pgn_file_path):
+        if pathlib.Path.exists(pgn_file_path):
+            self.pgn_file = open(pgn_file_path)
```

Comment:
RFC: a more declarative approach might be preferable if this class grows, like

```python
class PgnFileLoader:
  pgn_file = None
  pgnDataInstance = None
```

Reply:
background is that if this class grows you'll most probably lose overview of what variables exist, like this it can't happen

Reply:
Makes sense 👌🏽

It's funny, I had it written like you suggested and then took it off.

Will address requests